### DISCUSSION TITLE
⛽ Fueling Collaboration & Knowledge Exchange - Copilot 🤖  Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#57420] [Copilot News and Announcements]
### `Life is better with company - every Developer needs Copilot  `🧑‍✈️ 

👋 We’re back with the latest from the world of AI 🌎 powered by GitHub Copilot which started a new age of software development as an AI pair programmer, in less than two years since its launch, GitHub Copilot has [already improved](https://github.blog/2023-03-22-github-copilot-x-the-ai-powered-developer-experience/) writing 46% of code written and helped developers code up to 55% faster. 🚀

Moving forward, I’ll share separate discussions for important announcements, events, and feature releases. Keep reading our bi-weekly check-ins, where we'll bring it all together.

## 🤝  Help us build your Copilot know-how [Join the FAQ](https://github.com/orgs/community/discussions/47318)❓

Do you encounter questions or challenges while using Copilot? We invite you to [join the discussion](https://github.com/orgs/community/discussions/47318) and contribute your own queries and common issues to help us create a valuable resource filled with essential Copilot FAQs and answers! 

Your Feedback Matters! For instance, we recently compiled a [Copilot Labs FAQ](https://github.com/orgs/community/discussions/55320) in response to numerous inquiries we received last month. This month we are hearing a lot about Copilot Chat from you. 

We already have a ton of questions [in the FAQ](https://github.com/orgs/community/discussions/47318). Let us together build an evergreen reference that addresses your concerns and empowers the community. Let's join our hands in shaping a knowledge hub that ensures smooth sailing in your Copilot 🤖 endeavors!

## ✨ Community Champions: Amplifying the Resounding Shoutouts! 📢

- Thank You [@brettcurtis](https://github.com/brettcurtis) for coming back to their [discussion](https://github.com/orgs/community/discussions/56558) and sharing the hotfix that helped them run Copilot for CLI. 💗
- A big round of applause ✨ for [@RustingSword](https://github.com/RustingSword) for sharing their expertise with the community, answering how to[ increase the font size of Copilot Chat](https://github.com/orgs/community/discussions/55469).
- Sparkles✨ for [@sr4dev](https://github.com/sr4dev) for doubling down on the issue at a glance and enabling users [get past the Status Code 404](https://github.com/orgs/community/discussions/55386) with Copilot Chat 👏

## From Novice to Pro ⭐  Learn GitHub Copilot from [Microsoft Learn](https://learn.microsoft.com/en-us/training/modules/introduction-to-github-copilot/) 

By harnessing AI, Copilot provides code suggestions and auto-completion based on the context of the code you are writing. It's like having a coding partner that can understand what you are trying to accomplish and helps you get there faster. 

In the [Microsoft Learn module](https://learn.microsoft.com/en-us/training/modules/introduction-to-github-copilot/), you'll learn how to set up [GitHub Copilot](https://github.com/features/copilot), understand how it works, and get started with using it in your own projects. 

## Resources to bookmark 🔖

- [Tips for building software with generative AI](https://github.com/orgs/community/discussions/56349)
- Previously - [Dive into the AI Frontier 🤖 [Copilot Community Check-in ] ](https://github.com/orgs/community/discussions/55823) 
- [How generative AI is making a difference](https://github.com/orgs/community/discussions/54334) - Developer Productivity and Satisfaction
- [Artificial Intelligence](https://github.blog/tag/artificial-intelligence/)
- [Generative AI](https://github.blog/tag/generative-ai/)
- [GitHub Copilot](https://github.blog/tag/github-copilot/)  
- [GitHub Copilot X](https://github.blog/tag/github-copilot-x/)

Do you have a favorite feature or trick that you love using with Copilot? We would love to hear about it! Share your thoughts in the comments below. Happy coding! 🚀

**Comment by diogofcunha** [comment]
Hey @Akash1134 thanks for you contributions to this community.

I'm looking for the best place to enquire something that has been on my mind (and probably a lot of people in similar positions).

I am keen to understand how organisations are typically tracking and quantifying the efficiency improvements introduced by the usage of GitHub Copilot. As an engineering manager, I am eager to explore all available avenues that could potentially enhance our development processes and productivity and I would love to find out more about the statement _"writing 46% of code written and helped developers code up to 55% faster"_.

It's clear that the team has done an extraordinary job here and that the tool has potential for boosting productivity and speeding up development cycles, but measuring these improvements in a concrete way seems to be a less straightforward task.

If you could provide some insights or even share resources about the best practices organisations are following to track these improvements, it would be greatly appreciated. For instance:

1. Are there specific metrics that companies tend to monitor?
2. What kind of data do they collect before and after the adoption of Copilot to assess its impact?
3. Are there any case studies or testimonials that quantify the benefits?

If i'm in the wrong space really sorry about that and if you could be kind enough to redirect me in that case it would be awesome.

Once again thank you for all the work you've been doing 🚀

**Reply by Akash1134** [reply]
Hi @diogofcunha ,

Thanks for the kind words. You are indeed at the right place to ask questions and I'll try to answer them one by one. 

> I am keen to understand how organisations are typically tracking and quantifying the efficiency improvements introduced by the usage of GitHub Copilot. As an engineering manager, I am eager to explore all available avenues that could potentially enhance our development processes and productivity and I would love to find out more about the statement "writing 46% of code written and helped developers code up to 55% faster".

Please read [this blog post ](https://github.blog/2023-05-09-how-companies-are-boosting-productivity-with-generative-ai/) to get a detailed overview of how we measured developer productivity and how Copilot has proven to enhance it. 

For more context on the statement "writing 46% of code written and helped developers code up to 55% faster" -  read this incredible blog on [Research: quantifying GitHub Copilot’s impact on developer productivity and happiness ](https://github.blog/2022-09-07-research-quantifying-github-copilots-impact-on-developer-productivity-and-happiness/)

 Specifically for Copilot for Business, [this detailed post](https://github.blog/2023-02-14-github-copilot-for-business-is-now-available/) will help align with our first stride in this future of AI for enterprises, a future that will push the boundaries for all developers. 

> Are there specific metrics that companies tend to monitor? 
What kind of data do they collect before and after the adoption of Copilot to assess its impact?
Are there any case studies or testimonials that quantify the benefits?

Again, this post is a perfect read to get a clear understanding of all the three questions: [Research: quantifying GitHub Copilot’s impact on developer productivity and happiness ](https://github.blog/2022-09-07-research-quantifying-github-copilots-impact-on-developer-productivity-and-happiness/)

Let me know if have any questions. I hope that this help you better assess Copilot to supercharge your team's productivity, development processes, and happiness. 🙂
